src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,7 +9,8 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should contain up to 40 alphanumeric characters";
+    public static final String MESSAGE_CONSTRAINTS =
```

Comment:
Should we reduce the maximum length of a tag to something shorter, e.g. 25? I don't foresee tags like `Should pay me back in credit in 2020` being common or useful.

Reply:
I dont see any issue with keeping it at 40 unless you say the GUI side is going to get destroyed by length 40 tags

Reply:
Nah it won't haha